src/app/components/ConfirmOrCancel/index.tsx

```diff
@@ -22,7 +22,7 @@ function ConfirmOrCancel({
       <div className="mb-4">
         <Button
           onClick={onConfirm}
-          label="Confirm"
```

Comment:
Yep it is defaulted to 'Confirm'.

Reply:
I mean it should default to Confirm and can be overwritten whereever ConfirmOrCancel is used.

Reply:
Yep that is the way it works now, the `label` prop on the `Button` component is assigned the `label` prop of the `ConfirmOrCancel` component. And the default value of that is 'Confirm', so if no `label` prop is passed into the `ConfirmOrCancel` component then it will use 'Confirm'. Otherwise you can assign the `label` prop and it will use whatever string you pass in.

Reply:
ok, perfect!